cats/__init__.py

```diff
@@ -119,22 +128,37 @@ def parse_arguments():
         "Template found at https://github.com/GreenScheduler/cats/blob/main/config.yml.",
     )
     parser.add_argument(
-        "--jobinfo",
+        "--profile",
         type=str,
-        help="Resources used by the job in question, used to estimate total energy usage and carbon footprint. "
-        "E.g. `cpus=2,gpus=0,memory=8,partition=CPU_partition`. Valid components are "
-        "`cpus`: number of CPU cores; `gpus`: number of GPUs; `memory`: memory available in GB; "
-        "`partition`: one of the partitions keys given in `config.yml`. "
-        "Default: if absent, the total carbon footprint is not estimated.",
+        help="Hardware profile, specified in configuration file",
     )
-
     parser.add_argument(
         "--format",
         type=str,
         help="Format to output optimal start time and carbon emmission"
         "estimate savings in. Currently only JSON is supported.",
         choices=["json"],
     )
+    parser.add_argument(
+        "-f",
+        "--footprint",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--cpu",
+        type=positive_integer,
+        help="Number of cpus used by the job",
+    )
+    parser.add_argument(
+        "--gpu",
+        type=positive_integer,
+        help="Number of cpus used by the job",
+    )
+    parser.add_argument(
+        "--memory",
```

Comment:
is the unit GiB?

Reply:
It's GB, because the number we have in constants.py (or currently fixed_parameters.yml is power consumption in W/GB).